src/renderers/webvr/WebVRManager.js

```diff
@@ -179,6 +180,29 @@ function WebVRManager( renderer ) {
 
 				}
 
+				// Grip
+
+				buttonId = gamepad.buttons.length > 2 ? 2 : 0;
```

Comment:
Can you explain this line?

Reply:
@mrdoob On all the controllers (unless daydream) index=1 is the main button/trigger (the one used on the `selectStart` and index=2 is the grip.
If the controller has enough buttons to have an index=2, let's use that as it will be the grip, otherwise use the index=0 that in the daydream controller will be sharing the same action as the selectstart/end, and on the other controllers will be using whatever button they have mapped on 0: on vive, oculus go/quest it's the trackpad/joystick.

Reply:
well it could also be `gamepad.buttons[2] ? 2 : 0`

Reply:
> otherwise use the index=0 that in the daydream controller will be sharing the same action as the selectstart/end

Uh... Is that what Daydream does with WebXR? Using the same button for select and for squeeze?

Reply:
Ok, I have just checked it on daydream and it does not behave like that, it just doesn't support `squeeze` events at all. So I updated the code to prevent emitting events if grip is not present (buttons[2] === undefined)